src/app/header/components/TermButtons.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+import React, { useCallback, useContext, useMemo } from 'react';
+
+import { useTerm } from '../../context/TermContext';
+import { getButtonTerms, getTerms } from '../shared/utils';
+
+export function TermButtons() {
+  const [term, setTerm] = useTerm();
```

Comment:
I'm not sure if we want to store `term` here. I would think maybe storing and setting it in a context would be good (since a lot of stuff uses `term`)

Reply:
Wait ignore me I misread.